package.json

```diff
@@ -14,12 +14,15 @@
     "prepublishOnly": "script/publish",
     "lint": "eslint . --ext .ts,.tsx,.js --fix",
     "build": "npm install && node script/barrelize.js && tsc && npm run copy",
+    "predoc": "rm -rf docs",
```

Comment:
Are `predoc` or `postdoc` used anywhere?

Reply:
predoc and postdoc are called before and after npm run doc. It is just npm convention.